susemanager-utils/susemanager-sls/susemanager-sls.changes

```diff
@@ -1,3 +1,5 @@
+- Ignore non-executable-script rpmlint ceheck for shell scripts
```

Comment:
No changelog needed in this case, as this is not relevant for users, and we already have another entry that will allow us to tag.

Reply:
Just dropped the commit with the changelog entry